staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -168,73 +168,67 @@ func (o *RolloutStatusOptions) Run() error {
 		LabelSelectorParam(o.LabelSelector).
 		FilenameParam(o.EnforceNamespace, o.FilenameOptions).
 		ResourceTypeOrNameArgs(true, o.BuilderArgs...).
-		SingleResourceType().
+		ContinueOnError().
```

Comment:
Could you please elaborate what will happen if one of resources get error like in here https://github.com/kubernetes/kubernetes/pull/108777/files#diff-484e3b9bfad2141dd92637f42b9d227b2d7e91d0932bb85ec7cf098c20770691R394-R396.

Do resources, whose are not getting any error, stay in waiting stage? or we exit immediately?

Reply:
We run against every visited resource as opposed to exiting immediately on the first error, e.g.

```
_output/bin/kubectl rollout status -f . -R --timeout=10s 
deployment "echoserver-1" successfully rolled out
Waiting for deployment "echoserver-2" rollout to finish: 0 of 1 updated replicas are available...
deployment "echoserver-3" successfully rolled out
error: timed out waiting for the condition
```

While looking into other `rollout` commands, this looks consistent but I noticed these commands (`pause, resume, history`) have a `Flatten()` call prior to `Do()`. This omission doesn't seem to change the output 🤔